TelCo.ColorCoder/Program.cs

```diff
@@ -1,147 +1,47 @@
-﻿using System;
-using System.Diagnostics;
+using System;
 using System.Drawing;
+using System.Diagnostics;
+using TelCo.ColorCoderFunctions;
+using TelCo.ColorCoderClasses;
 
-namespace TelCo.ColorCoder
+namespace TelCo.ColorCoderTests
 {
     /// <summary>
-    /// The 25-pair color code, originally known as even-count color code, 
-    /// is a color code used to identify individual conductors in twisted-pair 
-    /// wiring for telecommunications.
-    /// This class provides the color coding and 
-    /// mapping of pair number to color and color to pair number.
+    /// This class contains the test cases
     /// </summary>
-    class Program
+    public class ColorCoderTest
```

Comment:
Program.cs - conginitive compexity is high and Code Duplication exist.
Main method - hold multiple responsibility

Reply:
modified the Main method and tried to remove duplications